CMakeLists.txt

```diff
@@ -172,6 +174,8 @@ nn_check_sym (atomic_cas_32 atomic.h NN_HAVE_ATOMIC_SOLARIS)
 
 nn_check_struct_member(msghdr msg_control sys/socket.h NN_HAVE_MSG_CONTROL)
 
+check_c_compiler_flag ("-fPIC -Werror" NN_HAVE_FPIC)
+
```

Comment:
This starts a precedent I want to avoid.  The compiler flags knowledge can get really hairy -- especially with some of the flags for optimizations, different debugging formats, etc.  And there are reasonable compilers that have different flags for PIC.  E.g. SunPro uses -KPIC.  So, let's just use the cmake magic to enable this instead, please, rather than carrying this baggage around ourselves.